pantheon-bundle/frontend/src/app/contexts/ProductContext.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { createContext, useState, useEffect } from "react";
+
+export let ProductContext = createContext<IProduct[]>([] as IProduct[]);
+
+export interface IProductProviderProps {
+  children?: React.ReactNode;
+}
+
+export interface IProduct {
+  description: string,
+  isOpen: boolean,
+  ["jcr:created"]: number,
+  ["jcr:createdBy"]: string,
+  ['jcr:lastModified']: number,
```

Comment:
"jcr:lastModified" is a type Date. e.g
"jcr:lastModified": "Mon Sep 28 2020 11:20:19 GMT+0000",